scripts/update-yaml.py

```diff
@@ -13,9 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import yaml
 import argparse
 import errno
-import yaml
```

Comment:
I seem to have missed the discussion of the approach. Is that still on-going or can we fix this up and resolve this issue?